src/compas_timber/ghpython/components/CT_Model/code.py

```diff
@@ -24,6 +24,9 @@
 
 
 class ModelComponent(component):
+    def __init__(self):
+        self.model = TimberModel()
```

Comment:
wouldn't this just create an empty `TimberModel` every time the component is run? shouldn't the beams and joint etc. be added anyways (even if geometry is not calculated)

Reply:
I added this so that every the model would persist, even if the inputs changed. I don't know of another way to do this.

Reply:
Ok the other way would be to have the scene be persistent... you think that would be better?